runtime/lua/vim/lsp/protocol.lua

```diff
@@ -620,6 +620,13 @@ function protocol.make_client_capabilities()
         -- Send textDocument/didSave after saving (BufWritePost)
         didSave = true;
       };
+      codeAction = {
+        dynamicRegistration = false;
+
+        codeActionLiteralSupport = {
+          codeActionKind = {};
```

Comment:
```suggestion
            codeActionKind = {
                valueSet = {};
            };
```

I think the `valueSet` would still have to be set if it's empty. But given that it's not necessary I think it could be left out completely for now.